### ISSUE TITLE
ERL-1377: website downloads fail with newest chrome

### ISSUE THREAD

**Issue by OTP-Maintainer** [erlang/otp#4370]
Original reporter: `JIRAUSER16810`
Affected version: `Not Specified`
Component: `Not Specified`
Migrated from: https://bugs.erlang.org/browse/ERL-1377

---

```
Newer chrome versions don't allow http:// downloads of .exe files and hence block downloads of the windows distribution of erlang from:

[https://www.erlang.org/downloads]

E.g. the current link is: [http://erlang.org/download/otp_win64_23.1.exe]

which is blocked by Chrome: [https://blog.chromium.org/2020/02/protecting-users-from-insecure.html] since October

 

*Fix:*

Just update your download links to be using *https://* instead of current http://

 
```

**Comment by aldobongio** [comment]
The official [download pag](https://www.erlang.org/downloads)e is still broken: the Windows exe links are exposed with `http `instead of `https`.

**Comment by vkatsuba** [comment]
In web console:
```
downloads:1 Mixed Content: The site at 'https://www.erlang.org/' was loaded over a secure connection, but the file at 'http://erlang.org/download/otp_win32_24.1.exe' was loaded over an insecure connection. This file should be served over HTTPS. This download has been blocked. See https://blog.chromium.org/2020/02/protecting-users-from-insecure.html for more details.
```

**Comment by KennethL** [comment]
Should be fixed for OTP 24.1 now.
